webapp/webpack.config.js

```diff
@@ -0,0 +1,42 @@
+var path = require('path');
+
+module.exports = {
+    entry: [
+        './src/index.js',
+    ],
+    resolve: {
+        modules: [
+            'src',
+            'node_modules',
+        ],
+        extensions: ['*', '.js', '.jsx'],
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['env', 'react'],
+                        plugins: [
+                            'transform-class-properties',
+                            'transform-object-rest-spread',
+                        ],
+                    },
+                },
+            },
+        ],
+    },
+    externals: {
+        react: 'react',
+        redux: 'redux',
+        'react-redux': 'reactRedux',
```

Comment:
This depends on a forthcoming webapp change to export not `react-redux` but `reactRedux`. The way Webpack externals work, it basically tries to evaluate the exported name (hitting `window` given the global context), but `react-redux` literally resolves to `NaN`.